ql/src/test/results/clientpositive/llap/input_part1.q.out

```diff
@@ -22,7 +22,7 @@ POSTHOOK: Input: default@srcpart@ds=2008-04-08/hr=12
 POSTHOOK: Output: default@dest1_n45
 OPTIMIZED SQL: SELECT `key`, `value`, CAST('12' AS STRING) AS `hr`, CAST('2008-04-08' AS STRING) AS `ds`
 FROM `default`.`srcpart`
-WHERE `key` < 100 AND `ds` = '2008-04-08' AND `hr` = '12'
+WHERE `key` < 100 AND (`ds` = '2008-04-08' AND `hr` = '12')
```

Comment:
Some of these conditions are not flattened anymore. Not sure about the root cause.